### PR TITLE
Add new product pages

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
## First part: translate 1 page (CAPP LMS)
- [x] Fix capp-lms localization and add keys for i18n
- [x] Localize content partials
- [x] Import hero partial
- [x] Redirect old path
- [x] Remove capp-entwickeln (incl keys)
- [x] Update solutions homepage
- [x] Update hero homepage
- [x] Update product index (CAPP LMS and remove capp-bilden) 
- [x] Remove all now unused assets
- [x] Fix all internal links (navigation and all mentions of capp-bilden and capp-entwickeln)
- [x] Remove old productsheets (+ mentions)

## Second part: More Product Pages
- [x] Update navigations (Main, Mobile, Footer)
- [x] Fix all links to Learning Management page
- [x] Add real 🇩🇪content
  - [x] CAPP Learning Management page 
  - [x] Talentmanagement page
  - [x] Compliance page
  - [x] Homepage
    - [x] Hero
    - [x] Solutions
  - [x] Product index
- [x] Update paths
- [x] Links on CAPP Learning Management page
- [x] Screenshots @lennartdefacto 
- [x] Image alts @lennartdefacto 
- [x] Icons
- [x] Custom titles on the card forms
- [x] Fit Qualifikationsmanagement in parents
- [x] Deploy to staging

Close #1496

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
This just needs new content, including assets like images.

**Comment by RonaldDijkstra** [comment]
Any progress on content here @lennartdefacto?

**Comment by RonaldDijkstra** [comment]
Any progress on content here @lennartdefacto?

**Comment by RonaldDijkstra** [comment]
Needs rebase.

**Comment by RonaldDijkstra** [comment]
And it's rebased. Conflicts are gone.

**Comment by RonaldDijkstra** [comment]
In a follow up of this it would be pretty easy to address #1496 and after that new items and all that.

File: source/localizable/home/_hero.erb [hunk 1]
```diff
@@ -2,24 +2,29 @@
...
-        <%= image_tag "logos/capp-new-version-white.svg", alt: "CAPP LMS", class: "logo" %>
-        <h1><%= t("home.hero_capp.title") %></h1>
-        <h2><%= t("home.hero_capp.text") %></h2>
-        <%= link_to t("actions.read_more_2"), "/capp-lms.html", class: "button pink" %>
-        <%= link_to t("actions.home.demo"), "/demo-aanvragen.html", class: "button white-pink" %>
+        <h1>
+          <%= t("home.hero.title") %>
+        </h1>
+
+        <%= link_to t("actions.home.demo"), "/demo-aanvragen.html", "data-event" => "home_demo", class: "button" %>
```
Comment (sn3p) [inline source/localizable/home/_hero.erb]:
```suggestion
        <%= link_to t("actions.home.demo"), "/demo-aanvragen.html", "data-event":  "home_demo", class: "button" %>
```

File: source/blog/de/2016-11-03-schon-wieder-ein-neues-system.html.markdown [hunk 2]
```diff
@@ -8,7 +8,11 @@ image: images/blog/social/ein-neues-system-1200x630.png
 featured_image: images/blog/featured/20161103-system.png
 ---
 
-__Wir sind uns bewusst über die Tatsache, dass die Einführung eines [Lernmanagentsystems (LMS)](/capp-bilden/) viele Veränderungen in einer Organisation mit sich bringt. Ein solches System hat an sich schon einiges an Gewicht, aber dann kommt auch noch ein neues „Computersystem“ dazu.__
+<<<<<<< HEAD
+__Wir sind uns bewusst über die Tatsache, dass die Einführung eines [Lernmanagentsystems (LMS)](/capp-lms/) viele Veränderungen in einer Organisation mit sich bringt. Ein solches System hat an sich schon einiges an Gewicht, aber dann kommt auch noch ein neues „Computersystem“ dazu.__
+=======
+***Wir sind uns bewusst über die Tatsache, dass die Einführung eines [Lernmanagentsystems (LMS)](/capp-lms/) viele Veränderungen in einer Organisation mit sich bringt. Ein solches System hat an sich schon einiges an Gewicht, aber dann kommt auch noch ein neues „Computersystem“ dazu.***
+>>>>>>> Change CAPP Bilden to CAPP LMS
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/blog/de/2016-11-03-schon-wieder-ein-neues-system.html.markdown]: Thanks.

**Comment by RonaldDijkstra** [comment]
Goes in a different direction I think: #1356

Text for 3 different pages we do not have right now.

**Comment by RonaldDijkstra** [comment]
Already deployed to https://staging.defacto.nl for checking now (but still have some work todo)

**Comment by RonaldDijkstra** [comment]
@lennartdefacto I think you can add the right screenshots in.

**Comment by sn3p** [comment]
This PR is soooooooooo huge 😭

**Comment by RonaldDijkstra** [comment]
> This PR is soooooooooo huge 😭

It is... 🔪 🔪 🔪

**Comment by RonaldDijkstra** [comment]
> create a css box shadow option for screenshots, prevents manual work in the future, will speed up future updates of screenshots

#1671

**Pull request closed without merging**